roles/openshift_examples/examples-sync.sh

```diff
@@ -5,8 +5,8 @@
 #
 # This script should be run from openshift-ansible/roles/openshift_examples
 
-XPAAS_VERSION=ose-v1.2.0-1
-ORIGIN_VERSION=v1.1
+XPAAS_VERSION=master
```

Comment:
That's fine.  I guess we need to tell them to create the tag for the 1.2 release then.  I didn't see it when I looked.